components/threading/global_threads/global_thread.stories.tsx

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import moment from 'moment-timezone';
+import {Unit} from '@formatjs/intl-relativetimeformat';
+
+import {storiesOf} from '@storybook/react';
+import {withKnobs, boolean, number, select, text} from '@storybook/addon-knobs';
+import {action} from '@storybook/addon-actions';
+
+import StoryGrid from 'storybook/story_grid';
+import StoryBox from 'storybook/story_box';
+
+import ThreadItem from './thread_item';
+
+const units: Unit[] = [
+    'second',
+    'minute',
+    'hour',
+    'day',
+    'week',
+    'month',
+    'quarter',
+    'year',
+];
+
+const users = [
+    {
+        url: '/api/v4/users/1/image?_=0',
```

Comment:
Same here, add the `.` at the beginning to make it work locally and in the CI deployed version.